addons/point_of_sale/static/src/js/models.js

```diff
@@ -15,6 +15,7 @@ var QWeb = core.qweb;
 var _t = core._t;
 var round_di = utils.round_decimals;
 var round_pr = utils.round_precision;
+var Markup = utils.Markup
```

Comment:
This can be introduced as `const` since it'll affect master as well. Refactoring `var` to `const` can be tricky since the scope isn't the same and it'll take more time for nothing